privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -321,13 +321,13 @@ <h3 translate>Action</h3>
                     </label>
                     <label for="toggleAccordion">
                         <button class="btn btn-sm btn-success"
-                                ng-show="!isOpen"
-                                ng-click="isOpen = true">
+                                ng-show="groupsOpen < actionGroups.length"
```

Comment:
there is a little bug if You apply a filter and then change the `Scope` of the policy (which loads a different set of actions). The `close all groups` button does not work anymore.